### PR TITLE
changes collection path to lower case when removing collection

### PR DESCRIPTION
**Pull request by tsop14**
Fixes broken remove collection - which was not working for any collection with a capital letter. 

In remove function, changes collection path to lowercase.

### CODE REVIEW

**Comment by tsop14** [comment]
@jamesgorrie

**Comment by jamesgorrie** [comment]
This will fix the current issue - it'd be worth going through making sure nothing else was affected.
:+1:

File: collections/app/store/CollectionsStore.scala [hunk 1]
```diff
@@ -27,7 +27,7 @@ object CollectionsStore {
   }
 
   def remove(collectionPath: List[String]): Future[Unit] = {
-    val path = CollectionsManager.pathToString(collectionPath)
+    val path = CollectionsManager.pathToString(collectionPath).toLowerCase
```
Comment (kenoir) [inline collections/app/store/CollectionsStore.scala]: Should this be happening inside CollectionsManager?

Comment (jamesgorrie) [inline collections/app/store/CollectionsStore.scala]: Yup.

Comment (tsop14) [inline collections/app/store/CollectionsStore.scala]: 😞 making new PR for this now.

**Pull request merged**